### PR TITLE
[MO] Deprecate TensorFlow 1.x environment support by MO

### PR DESCRIPTION
**Pull request by achetver**
Remove tf1 requirements file and add deprecation message then tf1 used while compression.

There was a deprecation notice in the Release notes before even if it was not stated explicitly in the warning from code. There is a still possibility to use TF1 environment but starting from this release it will no longer be recommended and supported by OpenVINO. TF1 models should be inferred through TF2 compatibility mode.

Ticket: 68386, 72575

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names

Validation:
* [x]  Unit tests: N/A - No new implemented functions;
* [x]  Framework operation tests: N/A - No new enabled operations;
* [x]  Transformation tests: N/A - No new implemented transformations;
* [x]  Model Optimizer IR Reader check: N/A.

Documentation:
* [x]  Supported frameworks operations list: N/A - No new supported operations;
* [x]  Supported **public** models list: N/A - Nothing to add;
* [x]  User guide update: Cone snippets updated.

### CODE REVIEW

**Comment by achetver** [comment]
@mvafin @sadolini @yekruglov Please, take a look at these changes

**Review by sadolini — COMMENTED** [review]
Should we remove install_prerequisites_tf.sh/bat files? Or add some note inside, that tf1.x is deprecated. 
Should we remove requirements_tf mentions from docs?
And minor note, just to think about it. We have descriptions how to freeze model in docs, will it work in TF2.x enviroment? Or we assume that user will still run it on TF1.x?

File: model-optimizer/automation/package_BOM.txt [hunk 1]
```diff
@@ -1101,4 +1101,3 @@ requirements_kaldi.txt
 requirements_mxnet.txt
 requirements_onnx.txt
 requirements_tf.txt
-requirements_tf2.txt
```
Comment (undef-nnov) [inline model-optimizer/automation/package_BOM.txt:1104]: Quick grep for "requirements_tf2" shows several places in the documentation where it is mentioned:

```
bash-3.2$ grep -Isrn requirements_tf2 ./
.//MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md:339:In order to convert TensorFlow* 2 models, installation of dependencies from `requirements_tf2.txt` is required.
.//MO_DG/prepare_model/Config_Model_Optimizer.md:139:pip3 install -r requirements_tf2.txt
```

File: model-optimizer/mo/utils/versions_checker.py [hunk 2]
```diff
@@ -250,16 +250,14 @@ def check_requirements(framework=None):
...
         framework_suffix = ""
     elif framework == "tf":
-        if "tensorflow" in env_setup and env_setup["tensorflow"] >= LooseVersion("2.0.0"):
-            framework_suffix = "_tf2"
-        else:
-            framework_suffix = "_tf"
-    else:
-        framework_suffix = "_{}".format(framework)
+        if "tensorflow" in env_setup and env_setup["tensorflow"] < LooseVersion("2.0.0"):
+            log.error('\t\nTensorFlow1 usage detected. Since OV 2022.1 release it`s usage is deprecated.\n',
```
Comment (undef-nnov) [inline model-optimizer/mo/utils/versions_checker.py:259]: Expand "OV" to "OpenVINO"

File: model-optimizer/requirements_tf.txt [hunk 3]
```diff
@@ -1,7 +1,6 @@
-# TensorFlow 1.x and 2.x are incompatible, use separate virtual environments for each version
```
Comment (undef-nnov) [inline model-optimizer/requirements_tf.txt:1]: The deprecation process works differently. We first add a warning message that something is deprecated but still allow user to use it and then in next (or even the next after the next) release we remove possibility to use the deprecated feature. Here we completely remove the ability to install TF1 from requirements file without notice period. I believe that some customers may have installation scripts which configure the environment and use these files. May be we should consider more graceful transition?

CC @jane-intel , @andrei-kochin , @rkazants .

File: model-optimizer/setup.py [hunk 4]
```diff
@@ -111,8 +111,7 @@ def find_package_modules(self, package, package_dir):
       'kaldi': read_text('requirements_kaldi.txt'),
       'mxnet': read_text('requirements_mxnet.txt'),
       'onnx': read_text('requirements_onnx.txt'),
-      'tensorflow': read_text('requirements_tf.txt'),
-      'tensorflow2': read_text('requirements_tf2.txt'),
+      'tensorflow2': read_text('requirements_tf.txt'),
```
Comment (undef-nnov) [inline model-optimizer/setup.py:114]: Inconsistency between the name and the configuration file. Maybe keep both keys "tensorflow" and "tensorflow2" but install requirements from the same file?

File: model-optimizer/requirements_tf.txt (hunk 3 above)
Comment (andrei-kochin) [reply model-optimizer/requirements_tf.txt:1]: @undef-nnov There was a deprecation notice in the Release notes before even if it was not stated explicitly in the warning from code. There is a still possibility to use TF1 environment but starting from this release it will no longer be recommended and supported by OpenVINO. TF1 models should be inferred through TF2 compatibility mode.

File: model-optimizer/mo/utils/versions_checker.py (hunk 2 above)
Comment (achetver) [reply model-optimizer/mo/utils/versions_checker.py]: Done.

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 5]
```diff
@@ -124,19 +116,15 @@ virtualenv -p /usr/bin/python3.6 .env3/bin/activate
...
 
     *   To install dependencies only for Caffe:
 ```shell
 pip3 install -r requirements_caffe.txt
-```
-    *   To install dependencies only for TensorFlow 1.x:
-```shell
-pip3 install -r requirements_tf.txt
 ```
     *   To install dependencies only for TensorFlow 2.x:
```
Comment (achetver) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md:125]: Remove `only`

File: model-optimizer/install_prerequisites/install_prerequisites.bat [hunk 6]
```diff
@@ -54,7 +54,7 @@ IF /I "%1%" EQU "" (
         set postfix=_tf
```
Comment (achetver) [inline model-optimizer/install_prerequisites/install_prerequisites.bat:54]: Check do we need a remove tf from here

File: model-optimizer/automation/package_BOM.txt (hunk 1 above)
Comment (achetver) [reply model-optimizer/automation/package_BOM.txt:1103]: Now it's removed

File: model-optimizer/setup.py (hunk 4 above)
Comment (achetver) [reply model-optimizer/setup.py]: Done.

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 5 above)
Comment (sadolini) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md:125]: I don't think only should be removed, it looks ok: dependencies only for caffe, only for mxnet,... Possibly add notice that TF 2 will be installed because tf1.x support deprecated.

**Review by sadolini — COMMENTED** [review]
Do we check that MO with tf2 can load non-freezed models from tf1 (timestamp, metagraph etc)?  If it will not work additional notes are needed in docs.

**Comment by achetver** [comment]
> Do we check that MO with tf2 can load non-freezed models from tf1 (timestamp, metagraph etc)? If it will not work additional notes are needed in docs.

I remember your point and will check it :)

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md [hunk 7]
```diff
@@ -336,7 +336,7 @@ TensorFlow*-specific parameters:
 
 ## Convert TensorFlow* 2 Models <a name="Convert_From_TF2X"></a>
 
-In order to convert TensorFlow* 2 models, installation of dependencies from `requirements_tf2.txt` is required.
+In order to convert TensorFlow* 2 models, installation of dependencies from `requirements_tf.txt` is required.
```
Comment (ilya-lavrenov) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md:339]: @slyubimt is the file name aligned with AC?

File: tools/mo/setup.py [hunk 8]
```diff
@@ -109,7 +109,7 @@ def find_package_modules(self, package, package_dir):
       'mxnet': read_text('requirements_mxnet.txt'),
       'onnx': read_text('requirements_onnx.txt'),
       'tensorflow': read_text('requirements_tf.txt'),
-      'tensorflow2': read_text('requirements_tf2.txt'),
+      'tensorflow2': read_text('requirements_tf.txt'),
```
Comment (ilya-lavrenov) [inline tools/mo/setup.py:112]: In general I don't understand why the file is renamed. Deprecation does not mean that user cannot use tf, it means that such support will be dropped in some time

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md (hunk 7 above)
Comment (slyubimt) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md:339]: There is no policy for naming requirements files right now. requirements-tensorflow.in used for `tensorflow2` extra tag for OMZ tools: https://github.com/openvinotoolkit/open_model_zoo/blob/ee423afb0d63d980b1d353dd240c5658483244cc/tools/model_tools/setup.py#L101

@jkamelin , @eaidova Do you have any preferences for this?

File: tools/mo/install_prerequisites/install_prerequisites.sh [hunk 9]
```diff
@@ -28,7 +28,7 @@ for ((i=1;i <= $#;i++)) {
         *)
             if [[ "$1" != "" ]]; then
                 echo "\"${!i}\" is unsupported parameter"
-                echo $"Usage: $0 {caffe|tf|tf2|mxnet|kaldi|onnx} {venv}"
```
Comment (slyubimt) [inline tools/mo/install_prerequisites/install_prerequisites.sh:31]: I believe we should keep `tf2` option in order to escape confusion for those users are used to using it in previous versions.

Comment (achetver) [reply tools/mo/install_prerequisites/install_prerequisites.sh]: Moved back.

File: tools/mo/setup.py (hunk 8 above)
Comment (achetver) [reply tools/mo/setup.py:112]: See comment above from Andrey Kochin.

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md (hunk 7 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md:339]: Now we have both requirements_tf2.txt and requirements_tf.txt with same requirements

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 5 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md:125]: Ok

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 10]
```diff
@@ -46,21 +46,13 @@ install_prerequisites_caffe.sh
...
 ```
-*   For TensorFlow\* 1.x on Linux:
-```
-install_prerequisites_tf.sh
-```
-*   For TensorFlow 1.x on Windows:
-```
-install_prerequisites_tf.bat
-```
 *   For TensorFlow\* 2.x on Linux:
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md:49]: please recover these lines, otherwise, it will be confusion as if TF1 models are no longer supported

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 11]
```diff
@@ -46,21 +46,13 @@ install_prerequisites_caffe.sh
...
 ```
-install_prerequisites_tf2.sh
+install_prerequisites_tf.sh
 ```
 *   For TensorFlow 2.x on Windows:
 ```
-install_prerequisites_tf2.bat
+install_prerequisites_tf.bat
 ```
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md:56]: the same comment

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 12]
```diff
@@ -124,19 +116,15 @@ virtualenv -p /usr/bin/python3.6 .env3/bin/activate
 ```shell
 pip3 install -r requirements.txt
 ```
-> **NOTE**: TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
+> **NOTE**: Since OpenVINO 2022.1 release TensorFlow1 usage is deprecated. TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md:120]: please correct the note. Now it is incorrect and absolutely confusing:
NOTE: Support of MO in TensorFlow 1.x environment is deprecated. Due to this `requirements_tf.txt` dependency is changed to TensorFlow 2.x. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models.

File: docs/install_guides/installing-openvino-linux.md [hunk 13]
```diff
@@ -240,14 +240,9 @@ cd /opt/intel/openvino_2022/tools/model_optimizer/install_prerequisites
...
-   - For **TensorFlow 1.x**:
-   ```sh
-   sudo ./install_prerequisites_tf.sh
-   ```
-
    - For **TensorFlow 2.x**:
    ```sh
-   sudo ./install_prerequisites_tf2.sh
+   sudo ./install_prerequisites_tf.sh
    ```
```
Comment (rkazants) [inline docs/install_guides/installing-openvino-linux.md:246]: please revert this lines

File: docs/install_guides/installing-openvino-macos.md [hunk 14]
```diff
@@ -213,14 +213,9 @@ Configure individual frameworks separately **ONLY** if you did not select **Opti
...
-   - For **TensorFlow 1.x**:
-   ```sh
-   sudo ./install_prerequisites_tf.sh
-   ```
-
    - For **TensorFlow 2.x**:
    ```sh
-   sudo ./install_prerequisites_tf2.sh
+   sudo ./install_prerequisites_tf.sh
    ```
```
Comment (rkazants) [inline docs/install_guides/installing-openvino-macos.md:219]: the same comment

File: docs/install_guides/installing-openvino-windows.md [hunk 15]
```diff
@@ -198,14 +198,9 @@ cd C:\Program Files (x86)\Intel\openvino_2022\tools\model_optimizer\install_prer
...
-   * For **TensorFlow 1.x**:<br>
-   ```sh
-   install_prerequisites_tf.bat
-   ```
-
    * For **TensorFlow 2.x**:<br>
    ```sh
-   install_prerequisites_tf2.bat
+   install_prerequisites_tf.bat
    ```
```
Comment (rkazants) [inline docs/install_guides/installing-openvino-windows.md:204]: the same comment

File: tools/mo/install_prerequisites/install_prerequisites.sh [hunk 16]
```diff
@@ -19,7 +19,7 @@ V_ENV=0
 
 for ((i=1;i <= $#;i++)) {
     case "${!i}" in
-        caffe|tf|tf2|mxnet|kaldi|onnx)
+        caffe|tf|mxnet|kaldi|onnx)
             postfix="_$1"
```
Comment (rkazants) [inline tools/mo/install_prerequisites/install_prerequisites.sh:23]: the same comment

File: tools/mo/install_prerequisites/install_prerequisites_tf2.bat [hunk 17]
```diff
@@ -5,4 +5,3 @@
 
 :: Check if Python is installed
 CALL install_prerequisites.bat tf2
```
Comment (rkazants) [inline tools/mo/install_prerequisites/install_prerequisites_tf2.bat:7]: please stay unchanged

File: tools/mo/install_prerequisites/install_prerequisites_tf2.sh [hunk 18]
```diff
@@ -4,4 +4,3 @@
 # SPDX-License-Identifier: Apache-2.0
 
 source install_prerequisites.sh tf2
```
Comment (rkazants) [inline tools/mo/install_prerequisites/install_prerequisites_tf2.sh:6]: please stay unchanged

File: tools/mo/requirements_tf.txt [hunk 19]
```diff
@@ -1,7 +1,6 @@
-# TensorFlow 1.x and 2.x are incompatible, use separate virtual environments for each version
-tensorflow~=1.15.5
+tensorflow~=2.5.0
 networkx~=2.5
-numpy>=1.16.6,<1.19
+numpy>=1.16.6,<1.20
 defusedxml>=0.7.1
 requests>=2.25.1
 fastjsonschema~=2.15.1
```
Comment (rkazants) [inline tools/mo/requirements_tf.txt:6]: please do not remove `requirements_tf2`. Let us just have two copies

File: tools/mo/openvino/tools/mo/utils/versions_checker.py [hunk 20]
```diff
@@ -251,16 +251,14 @@ def check_requirements(framework=None):
...
         framework_suffix = ""
     elif framework == "tf":
-        if "tensorflow" in env_setup and env_setup["tensorflow"] >= LooseVersion("2.0.0"):
-            framework_suffix = "_tf2"
-        else:
-            framework_suffix = "_tf"
-    else:
-        framework_suffix = "_{}".format(framework)
+        if "tensorflow" in env_setup and env_setup["tensorflow"] < LooseVersion("2.0.0"):
+            log.error('\t\nTensorFlow1 usage detected. Since OpenVINO 2022.1 release it`s usage is deprecated.\n',
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/utils/versions_checker.py:260]: Please write as follows:
```
Support of the Model Optimizer tool in TensorFlow 1.x environment is deprecated. It is highly recommended to use TensorFlow 2.x
```

Comment (rkazants) [reply tools/mo/openvino/tools/mo/utils/versions_checker.py:260]: @asomsiko, please recommend the warning sentence if TensorFlow 1.x usage is detected

**Review by rkazants — CHANGES_REQUESTED** [review]
please address comments and need feedback from @asomsiko

File: tools/mo/requirements_tf.txt (hunk 19 above)
Comment (achetver) [reply tools/mo/requirements_tf.txt:6]: 'requirements_tf2' file moved back

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 10 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md]: Done.

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 11 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md]: Done.

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 12 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md]: Note updated.

File: docs/install_guides/installing-openvino-linux.md (hunk 13 above)
Comment (achetver) [reply docs/install_guides/installing-openvino-linux.md]: Reverted.

File: docs/install_guides/installing-openvino-macos.md (hunk 14 above)
Comment (achetver) [reply docs/install_guides/installing-openvino-macos.md]: Done.

File: docs/install_guides/installing-openvino-windows.md (hunk 15 above)
Comment (achetver) [reply docs/install_guides/installing-openvino-windows.md]: Done.

File: tools/mo/install_prerequisites/install_prerequisites.sh (hunk 16 above)
Comment (achetver) [reply tools/mo/install_prerequisites/install_prerequisites.sh]: Done.

File: tools/mo/install_prerequisites/install_prerequisites_tf2.bat (hunk 17 above)
Comment (achetver) [reply tools/mo/install_prerequisites/install_prerequisites_tf2.bat]: Fixed.

File: tools/mo/install_prerequisites/install_prerequisites_tf2.sh (hunk 18 above)
Comment (achetver) [reply tools/mo/install_prerequisites/install_prerequisites_tf2.sh]: Fixed.

File: docs/install_guides/installing-openvino-linux.md [hunk 21]
```diff
@@ -240,11 +240,6 @@ cd /opt/intel/openvino_2022/tools/model_optimizer/install_prerequisites
...
 
-   - For **TensorFlow 1.x**:
-   ```sh
-   sudo ./install_prerequisites_tf.sh
-   ```
-
    - For **TensorFlow 2.x**:
    ```sh
    sudo ./install_prerequisites_tf2.sh
```
Comment (rkazants) [inline docs/install_guides/installing-openvino-linux.md:245]: please write it without particular indicated version as: 
```
For **TensorFlow**: 
sudo ./install_prerequisites_tf.sh
```

We will avoid confusion as if TF1 models no longer supported by MO

File: docs/install_guides/installing-openvino-macos.md [hunk 22]
```diff
@@ -213,11 +213,6 @@ Configure individual frameworks separately **ONLY** if you did not select **Opti
    sudo ./install_prerequisites_caffe.sh
    ```
 
-   - For **TensorFlow 1.x**:
-   ```sh
-   sudo ./install_prerequisites_tf.sh
-   ```
-
    - For **TensorFlow 2.x**:
```
Comment (rkazants) [inline docs/install_guides/installing-openvino-macos.md:216]: the same comment

File: docs/install_guides/installing-openvino-windows.md [hunk 23]
```diff
@@ -198,11 +198,6 @@ cd C:\Program Files (x86)\Intel\openvino_2022\tools\model_optimizer\install_prer
    install_prerequisites_caffe.bat
    ```
 
-   * For **TensorFlow 1.x**:<br>
-   ```sh
-   install_prerequisites_tf.bat
-   ```
-
```
Comment (rkazants) [inline docs/install_guides/installing-openvino-windows.md:205]: the same comment

File: tools/mo/requirements_tf.txt [hunk 24]
```diff
@@ -1,7 +1,6 @@
-# TensorFlow 1.x and 2.x are incompatible, use separate virtual environments for each version
-tensorflow~=1.15.5
+tensorflow~=2.5.0
 networkx~=2.5
-numpy>=1.16.6,<1.19
+numpy>=1.16.6,<1.20
```
Comment (rkazants) [inline tools/mo/requirements_tf.txt:3]: why is this change needed?

File: tools/mo/openvino/tools/mo/utils/versions_checker.py [hunk 25]
```diff
@@ -251,16 +251,15 @@ def check_requirements(framework=None):
...
-        if "tensorflow" in env_setup and env_setup["tensorflow"] >= LooseVersion("2.0.0"):
-            framework_suffix = "_tf2"
-        else:
-            framework_suffix = "_tf"
-    else:
-        framework_suffix = "_{}".format(framework)
+        if "tensorflow" in env_setup and env_setup["tensorflow"] < LooseVersion("2.0.0"):
+            log.error('\t\nSupport of the Model Optimizer tool in TensorFlow 1.x environment is deprecated.'
+                      'It is highly recommended to use TensorFlow 2.x.\n',
+                      extra={'is_warning': True})
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/utils/versions_checker.py:262]: @asomsiko, please check if we need to mention about security detail in this message.

**Review by rkazants — COMMENTED** [review]
Just one comment left. The other looks good. Please ask other reviewers

File: tools/mo/requirements_tf.txt (hunk 24 above)
Comment (achetver) [reply tools/mo/requirements_tf.txt:3]: Because here we copy dependency from tf2 file.

File: docs/install_guides/installing-openvino-linux.md (hunk 21 above)
Comment (achetver) [reply docs/install_guides/installing-openvino-linux.md]: Done.

File: docs/install_guides/installing-openvino-windows.md (hunk 23 above)
Comment (achetver) [reply docs/install_guides/installing-openvino-windows.md:205]: Done.

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 26]
```diff
@@ -453,15 +453,11 @@ virtualenv -p /usr/bin/python3.6 .env3/bin/activate
...
 
     *   To install dependencies only for Caffe:
 ```shell
 pip3 install -r requirements_caffe.txt
-```
-    *   To install dependencies only for TensorFlow 1.x:
-```shell
-pip3 install -r requirements_tf.txt
 ```
     *   To install dependencies only for TensorFlow 2.x:
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md:462]: To install dependencies only for TensorFlow

Comment (rkazants) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md:462]: and please correct `_tf2` -> `_tf`

Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md]: Done.

**Comment by achetver** [comment]
@mvafin @yekruglov @pavel-esir @sadolini Please, review these changes

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 27]
```diff
@@ -375,22 +375,14 @@ install_prerequisites_caffe.sh
 ```
 install_prerequisites_caffe.bat
 ```
-*   For TensorFlow\* 1.x on Linux:
+*   For TensorFlow\* on Linux:
 ```
 install_prerequisites_tf.sh
 ```
-*   For TensorFlow 1.x on Windows:
+*   For TensorFlow on Windows:
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md:382]: TensorFlow\*

Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md:382]: Looking to this file we don't have `*` after all frameworks names on windows and have it after all frameworks names on linux. I don't know the reason, but If you think it is necessary then we need to add `*` to all frameworks names on windows. What do you think?

**Review by sadolini — APPROVED** [review]

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 27 above)
Comment (jane-intel) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md:382]: @tsavina please help

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md [hunk 28]
```diff
@@ -29,6 +29,7 @@ by changing the `build_model_columns()` function in `census_dataset.py` as follo
 
 ```python
 def build_model_columns():
+  import tensorflow as tf
```
Comment (jane-intel) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md:32]: why? it is already imported in the original repo

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md [hunk 29]
```diff
@@ -59,36 +59,36 @@ xlnet_config = xlnet.XLNetConfig(json_path=XLNET_CONFIG_PATH)
 run_config = xlnet.RunConfig(is_training=False, use_tpu=False, use_bfloat16=False, dropout=0.1, dropatt=0.1,)
 
 
-sentence_features_input_idx = tf.placeholder(tf.int32, shape=[LENGTHS, BATCH], name='input_ids')
-sentence_features_segment_ids = tf.placeholder(tf.int32, shape=[LENGTHS, BATCH], name='seg_ids')
-sentence_features_input_mask = tf.placeholder(tf.float32, shape=[LENGTHS, BATCH], name='input_mask')
+sentence_features_input_idx = tf.compat.v1.placeholder(tf.int32, shape=[LENGTHS, BATCH], name='input_ids')
```
Comment (jane-intel) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md:62]: we use xlnet module here. does it work with tf2?

File: docs/install_guides/pypi-openvino-dev.md [hunk 30]
```diff
@@ -99,6 +99,7 @@ For example, to install and configure the components for working with TensorFlow
    ```sh
    pip install openvino-dev[tensorflow2,mxnet,caffe]
    ```
+**NOTE**: Support of MO in TensorFlow 1.x environment is deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models
```
Comment (jane-intel) [inline docs/install_guides/pypi-openvino-dev.md:102]: should we only have `pip install openvino-dev[tensorflow]` version for TF? Or `tensorflow2` is kept for backward compatibility?

**Review by jane-intel — DISMISSED** [review]

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md (hunk 29 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md:62]: From this page https://github.com/zihangdai/xlnet

> As of June 19, 2019, this code base has been tested with TensorFlow 1.13.1 under Python2.

So looks like no.

File: docs/install_guides/pypi-openvino-dev.md (hunk 30 above)
Comment (achetver) [reply docs/install_guides/pypi-openvino-dev.md:102]: We decide to have both for backward compatibility.

File: tools/mo/openvino/tools/mo/utils/versions_checker.py (hunk 25 above)
Comment (achetver) [reply tools/mo/openvino/tools/mo/utils/versions_checker.py:262]: @asomsiko Please, help us with this message.

File: docs/install_guides/pypi-openvino-dev.md (hunk 30 above)
Comment (jane-intel) [reply docs/install_guides/pypi-openvino-dev.md:102]: will we have them both forever?

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md (hunk 29 above)
Comment (jane-intel) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md:62]: Did you check that the code you've changed work for both tf1.x and tf2.x? I don't have enough confidence without your confirmation.

File: docs/install_guides/pypi-openvino-dev.md (hunk 30 above)
Comment (achetver) [reply docs/install_guides/pypi-openvino-dev.md:102]: I don't know. I think we need to adress this question to appropriate person.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md (hunk 29 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md:62]: Do we really need this check? If yes, then we need such checks for all other model modules/repositories and do checks for all future dependency changes.

Maybe is is better to have some message about Python/TensorFlow versions used in model repo?

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md (hunk 28 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md:32]: It was added during checks and can be removed.

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md (hunk 27 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/Config_Model_Optimizer.md:382]: @tsavina Please, take a look.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md (hunk 29 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md:64]: Note about original repository code added.

File: tools/mo/openvino/tools/mo/utils/versions_checker.py (hunk 25 above)
Comment (achetver) [reply tools/mo/openvino/tools/mo/utils/versions_checker.py:262]: @asomsiko Please, take a look.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md (hunk 28 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md]: Removed.

File: docs/install_guides/pypi-openvino-dev.md (hunk 30 above)
Comment (achetver) [reply docs/install_guides/pypi-openvino-dev.md:100]: Anyway we keep them for next release.

Comment (rkazants) [reply docs/install_guides/pypi-openvino-dev.md:100]: Let's proceed with both requirements files for the upcoming release

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
@azhogov, could you please merge it? Thanks
